_layouts/default.html

```diff
@@ -32,7 +32,7 @@ <h1><a class="extruded" href="/"><span>Ruby&nbsp;</span><span>Conferences</span>
         <ul>
           <li><a href="https://twitter.com/rubyconferences">@rubyconferences</a></li>
           <li><a href="https://github.com/ruby-conferences/ruby-conferences.github.io">source</a></li>
-          <li><a href="http://www.rubyconferences.org/calendar.ics">ics calendar feed</a></li>
+          <li><a href="https://www.rubyconferences.org/calendar.ics">ics calendar feed</a></li>
```

Comment:
@adarsh I think what we want is `https://rubyconferences.org/calendar.ics`

Reply:
I think we actually ended up with the http:// URL because the https:// with www didn't work, see https://github.com/ruby-conferences/ruby-conferences.github.io/issues/517#issuecomment-1716445151